### PR TITLE
Release 0.105.0

### PR DESCRIPTION
**Pull request by yuriolisa**
**Description:** <Describe what has changed.>

Release 0.105.0
**Link to tracking Issue(s):** <Issue number if applicable>

- Resolves: #3144 

**Testing:** <Describe what testing was performed and which tests were added.>

**Documentation:** <Describe the documentation added.>

### CODE REVIEW

**Review by TylerHelmuth — APPROVED** [review]

**Review by frzifus — APPROVED** [review]

**Comment by frzifus** [comment]
Wasnt the plan to remove the TA feature gate in 0.105.0?

cc @jaronoff97

**Comment by jaronoff97** [comment]
yes, i don't think we need it anymore because the escaping bug has been fixed

**Review by pavolloffay — APPROVED** [review]

**Comment by pavolloffay** [comment]
@frzifus shall we merge this before the release? https://github.com/open-telemetry/opentelemetry-operator/pull/3174

**Review by iblancasa — APPROVED** [review]

**Comment by pavolloffay** [comment]
@yuriolisa could you please rebase and include https://github.com/open-telemetry/opentelemetry-operator/pull/3176

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,28 @@
...
+  To enable this feature, you need to specify the `--openshift-create-dashboard` argument to the operator
+- `bundle`: Create an specific bundle for OpenShift environments. (#3054)
+- `collector`: Enabling ipFamilies and ipFamilyPolicy to be configured via OpenTelemetryCollector (#2958)
+- `auto-instrumentation`: Add Prometheus exporter to Python auto-instrumentation (#3122)
+
+### Components
+
+* [OpenTelemetry Collector - v0.105.0](https://github.com/open-telemetry/opentelemetry-collector/releases/tag/v0.105.0)
+* [OpenTelemetry Contrib - v0.105.0](https://github.com/open-telemetry/opentelemetry-collector-contrib/releases/tag/v0.105.0)
+* [Java auto-instrumentation - v1.32.1](https://github.com/open-telemetry/opentelemetry-java-instrumentation/releases/tag/v1.32.1)
```
Comment (pavolloffay) [inline CHANGELOG.md:19]: This needs to be updated https://github.com/open-telemetry/opentelemetry-operator/pull/3176

**Comment by yuriolisa** [comment]
> @yuriolisa could you please rebase and include #3176

I will rebase and submit the required changes.

File: CHANGELOG.md (hunk 1 above)
Comment (pavolloffay) [reply CHANGELOG.md:19]: also in versions.txt

Comment (yuriolisa) [reply CHANGELOG.md]: Just changed.

**Comment by jaronoff97** [comment]
@yuriolisa i just merged #3178, but we should wait until https://github.com/open-telemetry/opentelemetry-operator/pull/3174 is merged too

**Comment by jaronoff97** [comment]
@yuriolisa #3174 is merged, mind doing one last rebase and we can do the release?

**Comment by yuriolisa** [comment]
> @yuriolisa #3174 is merged, mind doing one last rebase and we can do the release?

Just sent.

**Review by jaronoff97 — APPROVED** [review]

**Review by swiatekm — APPROVED** [review]

**Pull request merged**